shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -96,7 +96,7 @@ - (instancetype)initWithPlugin:(NSString*)pluginKey flutterEngine:(FlutterEngine
 }
 
 - (NSView*)view {
-  return _flutterEngine.viewController.view;
+  return _flutterEngine.viewController.flutterView;
```

Comment:
here and elsewhere: I'm not clear on whether this is a concern for macOS, but on iOS we try to avoid to access the view without checking `viewLoaded` if there's any chance that the view is not already loaded into memory. This might not be the right spot (or the right PR to fix it here anyway), but down below I'll add another comment where it could be a problem.

Reply:
This seems to already being a problem before, what do you think the correct behavior is when FlutterEngine.view is called and the view is not loaded?

Reply:
For a11y we shouldn't force the view to load.  There are some instances here that are existing but we should just fix them.  Developers don't expect the view to get loaded this way.